### PR TITLE
facts - Set correct OS family for EuroLinux

### PR DESCRIPTION
**Pull request by jaromaz**
##### SUMMARY
This changes adds support for EuroLinux and adds it as a member to the RedHat family of operating systems.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
lib/ansible/module_utils/facts/system/distribution.py
lib/ansible/modules/hostname.py

### CODE REVIEW

**Comment by Akasurde** [comment]
@jaromaz Thanks for fixing this. Could you please take a look at unit tests that are failing https://dev.azure.com/ansible/ansible/_build/results?buildId=32565&view=logs&jobId=570d6be3-a35a-586c-0b02-6634bbb9f96c&j=40bacf7d-165d-5b57-35fc-8a6ffc0ddfca&t=92cdabea-8088-50d2-348b-96ccb91c1989 and fix them too?

File: changelogs/fragments/support_eurolinux.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - add EuroLinux to fact gathering
```
Comment (Akasurde) [inline changelogs/fragments/support_eurolinux.yml]:
```suggestion
  - distribution - add EuroLinux to fact gathering (https://github.com/ansible/ansible/pull/76624).
```

**Comment by jaromaz** [comment]
@Akasurde fixed :)

**Comment by Akasurde** [comment]
LGTM. Will wait for others to comment. Thanks.

**Review by bcoca — APPROVED** [review]

**Comment by Akasurde** [comment]
@jaromaz Thanks for the contribution. @bcoca thanks for the review.

**Pull request merged**